### PR TITLE
fix: swap colors in diff

### PR DESCRIPTION
**Pull request by wangrunji0408**
Signed-off-by: Runji Wang <wangrunji0408@163.com>

Fixes the order of diff contents so that the actual result is red and the expected result is green.

### CODE REVIEW

**Review by xxchan — COMMENTED** [review]
> actual result is red and the expected result is green

Why is that better? 😇 I think for diff, green means `+` and red means `-`.

**Comment by wangrunji0408** [comment]
But it's counterintuitive to show the right result in red. 😇

**Comment by xxchan** [comment]
Let's ask for others' opinion 😇 cc @skyzh

**Comment by skyzh** [comment]
As long as we tell developers that the left one is "actual" result and the right one is "expected" result it should be fine 😇

**Comment by xxchan** [comment]
But we don’t have left and right 😇

File: sqllogictest/src/runner.rs [hunk 1]
```diff
@@ -134,7 +134,7 @@ pub enum TestErrorKind {
         sql: String,
         err: Arc<dyn std::error::Error + Send + Sync>,
     },
-    #[error("query result mismatch:\n[SQL] {sql}\n[Diff]\n{}", difference::Changeset::new(.expected, .actual, "\n"))]
+    #[error("query result mismatch:\n[SQL] {sql}\n[Diff]\n{}", difference::Changeset::new(.actual, .expected, "\n"))]
```
Comment (xxchan) [inline sqllogictest/src/runner.rs:137]: What about
```suggestion
    #[error(
        "query result mismatch:\n[SQL] {sql}\n[Diff] (\x1b[91m-excepted\x1b[0m|\x1b[92m+actual\x1b[0m)\n{}",
        difference::Changeset::new(.expected, .actual, "\n").diffs.iter().format_with("\n", |diff, f| {
            match *diff {
                Difference::Same(ref x) => {
                    f(&format_args!(" {}", x))
                }
                Difference::Add(ref x) => {
                    f(&format_args!("\x1b[92m+{}\x1b[0m", x))
                }
                Difference::Rem(ref x) => {
                    f(&format_args!("\x1b[91m-{}\x1b[0m", x))

                }
            }
        })
    )]
```

image

Comment (wangrunji0408) [reply sqllogictest/src/runner.rs:137]: Look cool!

**Review by xxchan — APPROVED** [review]
I will merge and fix colors

**Pull request merged**